File: 08-20-listas/Main.java
import adts.LinkedList;
import adts.List;

public class Main {

  public static void main(String[] args) {
    List<Integer> l = new LinkedList<>();

    for (int i = 0; i < 10; i++) {
      l.insert(i);
    }

    for (int i = 0; i < 11; i++) {
      System.out.println(l.get(i));
    }
  }

}

File: 08-20-listas/main.cpp
#include "adts/linked_list.cpp"
#include <iostream>

int main()
{
  list<int> *l = new linked_list<int>();

  for (int i = 0; i < 10; i++)
  {
    l->insert(i);
  }

  for (int i = 0; i < 11; i++)
  {
    std::cout << l->get(i) << std::endl;
  }
}
